### DISCUSSION TITLE
Lesson 13:  Ganache server hostname drop down menu options have disappeared

### DISCUSSION THREAD

**Discussion by ChristianG77** [smartcontractkit/full-blockchain-solidity-course-py#331] [Q&A] [answered]
Hi,

I'm trying to deploy update_front_end.py using Ganache but it looks like the server hostname settings have changed and only give me one option 'HOSTNAME 0.0.0.0 - All Interfaces'.

![image](https://user-images.githubusercontent.com/91568178/140890489-27963279-c089-455c-99b5-8fbf0f53bdfa.png)

I used to have a WSL option for http://172.21.176.1:8545.

Anyone else encountering similar?

![image](https://user-images.githubusercontent.com/91568178/140890754-e153c259-c6ae-4b4f-8bb1-185bac18036c.png)

**Comment by cromewar** [comment, marked as answer]
@ChristianG77 Yes this happens due to a bug related with WSL, as you create a new host when you start a WSL terminal, ganache UI is upset because it can find that host, I hope the devs improve this for future releases, for now is a workaround to solve this.

first of all close ganache an then follow this steps:

1. on your keyboard hit "windows + r"
2. type `%appdata%`

![image](https://user-images.githubusercontent.com/20199180/140929297-014091d6-c7a3-4502-8b02-84cb29408c6c.png)

3. find the Ganache folder and delete it

![image](https://user-images.githubusercontent.com/20199180/140929387-bf009148-a77a-4483-9fb1-b582058e11de.png)

4. Start ganache again.

If you want no get rid off this, just remember to change the hostname in ganache after closing it, yes i know is a pain in the a** but is the only solution for now.

**Reply by ChristianG77** [reply]
Awesome that worked great, thank you!
